great_expectations/datasource/fluent/schemas/Datasource.json

```diff
@@ -0,0 +1,87 @@
+{
+    "title": "Datasource",
+    "description": "Base model for most fluent datasource related pydantic models.\n\nAdds yaml dumping and parsing methods.\n\nExtra fields are not allowed.\n\nSerialization methods default to `exclude_unset = True` to prevent serializing\nconfigs full of mostly unset default values.\nAlso prevents passing along unset kwargs to BatchSpec.\nhttps://docs.pydantic.dev/usage/exporting_models/",
```

Comment:
These descriptions are pulled from the class docstrings.
But because neither `Datasource` or `DataAsset` declare their own docstring we get the docstring from `FluentBaseModel`